ru.linux_gaming.PortProton.metainfo.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>ru.linux_gaming.PortProton</id>
```

Comment:
what is the relation between `linux-gaming.ru` and the upstream developers of this package?

Reply:
> what is the relation between `linux-gaming.ru` and the upstream developers of this package?

This is the website of the developer of the PortProton project

Reply:
this doesn't look obvious. Given that the URL in the appstream file do not point there.

Reply:
> this doesn't look obvious. Given that the URL in the appstream file do not point there.

Fixed